src/windows/file_properties.py

```diff
@@ -82,12 +82,8 @@ def __init__(self, file):
         file_extension = os.path.splitext(filename)[1]
         fps_float = float(self.file.data["fps"]["num"]) / float(self.file.data["fps"]["den"])
 
-        tags = ""
-        if "tags" in self.file.data:
-            tags = self.file.data["tags"]
-        name = filename
-        if "name" in self.file.data:
-            name = self.file.data["name"]
+        tags = self.file.data.get("tags", "")
+        name = file.data.get("name", filename)
```

Comment:
You dropped the 'self' there:
```suggestion
        name = self.file.data.get("name", filename)
```
(Or maybe I did, in my suggestion. Certainly possible.)

Reply:
Actually, I'm just gonna apply that (now that I've remembered I can do that), and then get this merged. Thanks again, @MartinThoma !